app/controllers/sign_up/completions_controller.rb

```diff
@@ -0,0 +1,24 @@
+module SignUp
+  class CompletionsController < ApplicationController
```

Comment:
This page is shown both to users who have signed up with LOA1 and users who have signed up with LOA3 so proofing should not be a pre-condition. I think logged in and `session[:sp]` should work